easybuild/easyconfigs/l/librsb/librsb-1.3.0.0-GCC-10.2.0.eb

```diff
@@ -0,0 +1,37 @@
+##
+# Author:    Michele Martone <michelemartone@users.sourceforge.net>
+##
+easyblock = 'ConfigureMake'
+
+name = 'librsb'
+version = '1.3.0.0'
+
+homepage = 'https://sourceforge.net/projects/librsb/'
+description = """A shared memory parallel sparse matrix computations library for the Recursive Sparse Blocks format"""
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['2ac8725d1f988f57df9383ae6b0bb2ed221ec935187d31ebb62ea95ee868a790']
+
+dependencies = [
+    ('zlib', '1.2.11')
+]
+
+# workaround to a configure defect manifesting during --rebuild: avoid mistaken detection of already existing librsb installation
+preconfigopts = "sed -i 's/export LIBRSB_CONFIG=.*$/export LIBRSB_CONFIG=no LIBRSB_CONFIG_VERSION=1.3.0;/g' configure*;"
+
+# workaround to a configure defect manifesting during --rebuild: ensure librsbpp/configure sees and uses root dir rsb.h
+local_cppflags = 'CPPFLAGS="${CPPFLAGS} -I%(builddir)s"'
+prebuildopts = local_cppflags
+preinstallopts = local_cppflags
```

Comment:
I found a very simple solution to those rebuild issues! All the trouble is caused by `$LIBRSB_CONFIG --I_opts` in `librsbpp/configure.ac`. That command will inject paths to the installation directory, which is emptied upon rebuild. Easy solution is to not use that command at all, delete those lines, regenerate the configure script and build as usual. I tested it and had no errors.

```suggestion
builddependencies = [
    ('Autotools', '20200321'),
]

dependencies = [
    ('zlib', '1.2.11')
]

# do not rely on `librsb-config --I_opts`, it will inject paths in install dir
preconfigopts = "sed -i '21,22d' librsbpp/configure.ac && "
preconfigopts += "autoreconf -fi &&"
```

Reply:
@boegel my understanding was that this will be properly fixed upstream by @michelemartone , so it should not be needed in future versions.